vnext/PropertySheets/Generated/PackageVersion.g.props

```diff
@@ -14,6 +14,6 @@
     <ReactNativeWindowsMajor>0</ReactNativeWindowsMajor>
     <ReactNativeWindowsMinor>70</ReactNativeWindowsMinor>
     <ReactNativeWindowsPatch>13</ReactNativeWindowsPatch>
-    <ReactNativeWindowsCanary>false</ReactNativeWindowsCanary>
+    <ReactNativeWindowsCommitId>1ed3be1c151cf86991ee3c566d11390c0c7f8b53</ReactNativeWindowsCommitId>
```

Comment:
Is the value a placeholder?
Is it updated on each release pipeline?

Reply:
it will be updated by each release pipeline
I chose for each branch what the previous run would have put in.